gnome-extra/evolution-data-server/metadata.xml

```diff
@@ -12,5 +12,6 @@
 			If gnome-online-accounts is enabled and used, this is not necessary,
 			but both can be supported at the same time with different setup at runtime</flag>
 		<flag name="weather">Enable optional weather calendar support</flag>
+		<flag name="phonenumber">Enable phone supports</flag>
```

Comment:
The rest of the <flag> tags are sorted by the name attribute, might as well keep the sorting